distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1164,6 +1173,52 @@ type Column
     day self = Value_Type.expect_has_date self <|
         self.make_unary_op "day"
 
+
+    ## Computes a time difference between the two dates.
+
+       It returns a column of integers expressing how many periods fit between
+       the two dates/times.
+
+       The difference will be positive if `end` is greater than `self`.
+
+       Arguments:
+       - end: A date/time column or a date/time value to compute the difference
+         from. It should have the same type as the current column, i.e. a
+         `Date_Time` column cannot be compared to a `Date` - to do so you first
+         need to `cast`.
+       - period: The period to compute the difference in. For `Date` columns it
+         should be a `Date_Period` and for `Time` columns it should be a
+         `Time_Period`. For `Date_Time` columns it can be either.
+    date_diff : (Column | Date | Date_Time | Time_Of_Day) -> Date_Period | Time_Period -> Column
+    date_diff self end (period : Date_Period | Time_Period) =
+        Value_Type.expect_type self .is_date_or_time "date/time" <|
+            my_type = self.inferred_precise_value_type
+            Value_Type.expect_type end (== my_type) my_type.to_display_text <|
+                Date_Time_Helpers.check_period_aligned_with_value_type my_type period <|
+                    new_name = self.naming_helpers.function_name "date_diff" [self, end, period.to_display_text]
+                    metadata = Operation_Metadata.Date_Period_Metadata.Value period my_type
+                    self.make_op "date_diff" [end] new_name metadata
+
+
+    ## Shifts the date/time by a specified period, returning a new date/time
+       column of the same type.
+
+       Arguments:
+       - amount: An integer or integer column specifying by how many periods to
+         shift each date.
+       - period: The period by which to shift. For `Date` columns it should be a
+         `Date_Period` and for `Time` columns it should be a `Time_Period`. For
+         `Date_Time` columns it can be either.
+    date_add : (Column | Date | Date_Time | Time_Of_Day) -> Date_Period | Time_Period -> Column
```

Comment:
```suggestion
    date_add : Integer | Column -> Date_Period | Time_Period -> Column
```

Since the docs for `amount` above say "integer or integer column".

Reply:
Right, it's an artifact from copying from `date_diff` - thanks for catching it!